linera-service/src/cli_wrappers.rs

```diff
@@ -742,25 +739,46 @@ impl LocalNetwork {
     }
 
     async fn run_server(&mut self, i: usize, j: usize) -> Result<Child> {
-        let storage = match self.database {
-            Database::RocksDb => format!("rocksdb:server_{}_{}.db", i, j),
-            Database::DynamoDb => format!("dynamodb:server_{}_{}.db:localstack", i, j),
-            Database::ScyllaDb => format!("scylladb:table_server_{}_{}_db", i, j),
+        let (storage, key) = match self.database {
+            Database::RocksDb => (format!("rocksdb:server_{}_{}.db", i, j), (i, j)),
+            Database::DynamoDb => (
+                format!("dynamodb:{}_server_{}.db:localstack", self.table_name, i),
+                (i, 0),
+            ),
+            Database::ScyllaDb => (
+                format!("scylladb:{}_server_{}_db", self.table_name, i),
+                (i, 0),
+            ),
         };
-        let key = (i, j);
         if !self.set_init.contains(&key) {
-            let mut command = self.command_for_binary("linera-server").await?;
-            command.arg("initialize");
-            if let Ok(var) = env::var(SERVER_ENV) {
-                command.args(var.split_whitespace());
+            let max_try = 4;
+            let mut i_try = 0;
+            loop {
+                let one_second = std::time::Duration::from_millis(1000);
+                std::thread::sleep(one_second);
```

Comment:
Is there a reason to sleep before the first try? Otherwise I'd move this to the end of the loop.

Reply:
Ok.